.travis.yml

```diff
@@ -44,7 +44,10 @@ before_install:
   - echo "build --worker_max_instances=2" >>~/.bazelrc
 
   # This is the magic incantation for making protobuf build 2x faster.
-  - echo "build --distinct_host_configuration=false" >>~/.bazelrc
+  # Unfortunately, it also breaks the build, due to a bug that has
```

Comment:
I'm inclined to just remove this entirely since from the issue it sounds like it's not really likely to be fixed any time soon and wasn't really recommended.  The slowness of building protobuf is probably something worth investigating afresh if we conclude it's still an issue.

Reply:
Okay, fine with me.